transmart-core-db/grails-app/domain/org/transmartproject/db/summaries/ConceptCount.groovy

```diff
@@ -0,0 +1,17 @@
+package org.transmartproject.db.summaries
+
+/**
+ * Created by piotrzakrzewski on 01/08/16.
+ */
+class ConceptCount {
```

Comment:
Unused, can be removed.

Reply:
done

Reply:
should be deleted after removing the old patient count logic. Although the logic uses sql, hibernate generates h2 tables from domain classes before running tests.